### ISSUE TITLE
Recipe editor item keys

### ISSUE THREAD

**Issue by MichelJonkman** [IoeCmcomc/MCDatapacker#19]
Just like what in the JSON happens, it'd be nice if we could define our keys and pattern separately. This would be especially useful if you have a lot of a single item in a recipe.

**Comment by IoeCmcomc** [comment]
If I understand correctly, you want to put the same item in multiple item slots. In this case, you can already drag-drop a item from a slot to another one while holding Ctrl to duplicate the item to the target slot. A normal drag-drop will move the item instead.
![MCDatapacker_K7y4FCkP7e](https://github.com/IoeCmcomc/MCDatapacker/assets/53734763/d4419ef8-e630-4f95-9922-49286a2b2034)

**Comment by MichelJonkman** [comment]
Aha yes this works, maybe a list of controls/shortcuts could be added for other people?

**Comment by IoeCmcomc** [comment]
I agree that the behaviors need to be documented. Maybe I will add cut/copy/paste command in the context menu (but not shortcuts due to technical limitations), so I will leave this issue open for now.

**Comment by IoeCmcomc** [comment]
I decided to add a status tip which display the following text when hovering an item slot: "Drag-drop to move item(s). Ctrl+Drag to duplicate item(s)".
